misc/config_tools/board_inspector/cli.py

```diff
@@ -7,7 +7,7 @@
 
 import sys, os
 import logging
-import subprocess
+import subprocess #nosec
```

Comment:
Can you verify that a space isn't required, ``#nosec`` vs. ``# nosec`` (the bandit documentation says to use a space).